code/WorkInProgress/JobXPRewards.dm

```diff
@@ -482,3 +482,53 @@ mob/verb/checkrewards()
 		banana.set_loc(get_turf(C.mob))
 		C.mob.put_in_hand(banana)
 		return
+
+/////////////Bartender////////////////
+
+/datum/jobXpReward/bartender
+	name = "Golden Cocktail Shaker"
+	desc = "After all your years of service, you've finally managed to gather enough money in tips to buy yourself a present! You regret every cent."
+	required_levels = list("Bartender"=0)
+	icon_state = "?"
+
+var/sacrifice_path = /obj/item/reagent_containers/food/drinks/cocktailshaker
+var/reward_path = /obj/item/reagent_containers/food/drinks/cocktailshaker/golden
+var/sacrifice_name = "cocktail shaker"
+
+activate(var/client/C)
+	var/O = locate(sacrifice_path) in C.mob.contents
+	if (istype(O, sacrifice_path))
+		boutput(C.mob, "You blink for a second and suddenly the cocktail shaker is covered from top to bottom in gold!")
+		var/obj/item/reagent_containers/food/drinks/cocktailshaker/golden/T = new/obj/item/reagent_containers/food/drinks/cocktailshaker/golden()
+		T.set_loc(get_turf(C.mob))
+		C.mob.put_in_hand(T)
+		return
+
+/*
+	activate(var/client/C)
+		var/found = 0
+		var/O = locate(sacrifice_path) in C.mob.contents
+		if (istype(O, sacrifice_path))
+			var/obj/item/gun/energy/egun/K = O
+			if (K.nojobreward) // Checks to see if it was scanned by a device analyzer
+				boutput(C.mob, "This [sacrifice_name] has forever been ruined by a device analyzer's magnets. It can't turn into a sword ever again!!")
```

Comment:
?

Reply:
Commented out code for WIP comparisons. Will remove when I can